.gitignore

```diff
@@ -2,3 +2,4 @@
 /flake.lock
 /.envrc
 /dist-newstyle/
+.pre-commit-config.yaml
```

Comment:
Is this file created on the fly?

Reply:
yeah, its created when you enter the nix shell